packages/onchainkit/src/identity/hooks/useAddress.ts

```diff
@@ -1,6 +1,6 @@
 import type {
   GetAddressReturnType,
-  UseAddressOptions,
+  UseAddressOptionsParams,
```

Comment:
nit: `OptionsParams` seems redundant. Couldn't we simplify to `UseAddressParams`?